### PR TITLE
documentation(EJ2-65258): Review corrections addressed for Message.

### PR DESCRIPTION
**Pull request by Deepika5698**
### Feature description

Need to include the review corrections for message feature list documentation.

### Analysis and design

- Add plural word (check the bold word) - In the below sample, the Message component content is customized with HTML elements and React Button components, which are directly added to the HTML element.
- To customize the Message component's appearance, add the custom class to the message through `cssClass` property. This custom class will be added to the root element. Based on this custom class, the user can override the styles of the message at the application level.
- Use the content in severity heading- The following example demonstrates the severities of the messages.
- Move the showIcon details to Icons md file under No Icon heading.
- Use the content- The variants of the message can be changed based on the variant property.
- Use the content-The following example demonstrates how the default message is rendered with a custom severity icon.
- Remove the custom appearance changes in custom icon heading sample. We will include it in customization heading.
- Maintain the following order in Icons md file.

   No Icon

   Custom Icon

   Close Icon

- Use the content- The Essential JS 2 Message has the following predefined CSS classes that can be defined in the HTML elements, which renders the message without any script reference.
- Remove the loading text from CSS Message heading. It has is displayed even displayed message content.
- Add reviewed content.

 
### Solution description
 
Added the reviewed corrections.
 
### Output screenshots
NA
 
### Areas affected and ensured
No area is affected due to this changes.
 
### Test cases
NA
 
### Test bed sample location
NA
 
### Additional checklist
 
* Did you run the automation against your implementation? NA
* Is there any API name change? No
* Is there any existing behavior change of other features due to this code change? No
* Does your new code introduce new warnings or binding errors? No
* Does your code pass all FxCop and StyleCop rules? Yes
* Did you record this case in the unit test or UI test? NA

### CODE REVIEW

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/icons.md [hunk 1]
```diff
@@ -10,45 +10,41 @@ documentation: ug
...
 
-## Close Icon
-
-The message can be rendered with or without the close icon. The close icon is used to hide the message, either by manually clicking the close icon or through keyboard interaction.
-
-By default, the close icon is not rendered in the message. To show the close icon, set the `showCloseIcon` property to `true`.
+## No Icon
 
-In the following example, the messages are rendered with the close icon.
+By default, severity icons can be displayed according to the severity types to make it more understandable to the user by visual information rather than text. To hide the severity icons, set the `showIcon` property to `false`.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/icons.md:17]: property should be in proper casing

**Review by PiramanayagamSyncfusion — COMMENTED** [review]
Changes fine

**Review by PiramanayagamSyncfusion — APPROVED** [review]

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/icons.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+The following example demonstrates the different severity messages without the severity icons.
+
+{% if page.publishingplatform == "aspnet-core" %}
+
+{% tabs %}
+{% highlight cshtml tabtitle="CSHTML" %}
+{% include code-snippet/message/severity/tagHelper %}
+{% endhighlight %}
+{% highlight c# tabtitle="Controller.cs" %}
+{% include code-snippet/message/severity/controller.cs %}
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/icons.md:28]: rename the sample name with no icon

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/severities.md [hunk 3]
```diff
@@ -10,13 +10,11 @@ documentation: ug
...
+The severity denotes the importance and context of the message to the user. The message contains different severity types. Use the `Severity` property to display the messages with different severity levels.
 
 The available severity types are **Normal**, **Success**, **Info**, **Warning** and **Error**. The default severity type for messages is **Normal**.
 
-The following example demonstrates the messages with different severity types.
-
-The following example demonstrates the messages with different severity types.
+The following example demonstrates the severity of the messages.
 
 {% if page.publishingplatform == "aspnet-core" %}
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/severities.md:20]: rename the sample name with severity

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/customization.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+{% include code-snippet/message/content-alignment/razor %}
+{% endhighlight %}
+{% endtabs %}
+{% endif %}
+
+![ASP.NET MVC Message Content Alignment](images/message-content-alignment.png)
+
+## Rounded and Square
+
+To customize the Message control's appearance, add the custom class to the message through the `cssClass` property. This custom class will be added to the root element. Based on this custom class, the user can override the message styles at the application level.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/customization.md:45]: need to use proper casing for `cssClass` property

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+description: Checkout and learn about template with ##Platform_Name## Message control of Syncfusion Essential JS 2 and more details.
+platform: ej2-asp-core-mvc
+control: Message
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
+# Template in Message control
+
+The message supports templates that allow you to customize the content with a custom structure. The content can be a string, paragraph, or any other HTML element. The template can be rendered through the `content` property or can be added directly to the `<e-content-template>` tags.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md:13]: Use below content

or added directly to the `<e-content-template>` tag.

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+In the following sample, the Message control content is customized with HTML elements and Syncfusion Button controls, which are directly added to the `content` property.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md:15]: Use below content
which are directly added to the `<e-content-template>` tag.

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/template.md [hunk 7]
```diff
@@ -10,9 +10,9 @@ documentation: ug
 
 # Template in Message control
 
-The message supports templates that allow you to customize the content with a custom structure. The content can be a string, paragraph, or any other HTML element. The template can be rendered through the `content` property or can be added directly to the HTML element.
+The message supports templates that allow you to customize the content with a custom structure. The content can be a string, paragraph, or any other HTML element. The template can be rendered through the `Content` property or added directly to the HTML element.
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/template.md:13]: Remove this content
or added directly to the HTML element

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/severities.md (hunk 3 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/severities.md:20]: Changes included in below pr
https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1117

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/icons.md (hunk 2 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/icons.md:28]: Changes included in below pr
https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1117

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md (hunk 5 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md:13]: Changes included in below pr
https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1117

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/template.md (hunk 7 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/template.md:13]: Changes included in below pr
https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1117

File: ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md (hunk 6 above)
Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.NETCORE/template.md:15]: Changes included in below pr
https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1117

**Pull request merged**